pom.xml

```diff
@@ -449,23 +449,6 @@
     <url>https://ci.openmicroscopy.org/</url>
   </ciManagement>
 
-  <mailingLists>
```

Comment:
Huh. No similar "forum" entry in https://maven.apache.org/pom.html#Environment_Settings

Reply:
Looking at how the Fiji crew has handled this https://github.com/fiji/fiji/blob/master/pom.xml#L145
Worth applying a similar change across all our repos?